aws/build-an-deploy-aws.sh

```diff
@@ -45,12 +39,13 @@ esac
 ACCOUNT_ID=$(aws sts get-caller-identity | jq '.Account' -r)
 echo "ACCOUNT_ID=${ACCOUNT_ID}"
 
+CLUSTERNAME="$(terraform output -raw cluster_name)"
+STATE_BUCKET="$(terraform output -raw state_bucket_name)"
+IRSA_ROLE_ARN="$(terraform output -raw irsa_role_arn)"
+EBS_ROLE_ARN="$(terraform output -raw ebs_role_arn)"
 
```

Comment:
can you please echo the configuration here? (e.g. the variables set) so we don't run into debug issues :) ?

Reply:
Done